android/gradle.properties

```diff
@@ -42,7 +42,7 @@ KEYSTORE_KEY_PASSWORD=password
 # By default we build a mostly universal APK
 ANDROID_ABI_SPLIT=false
 # Some platforms are excluded though
-ANDROID_ABI_INCLUDE=armeabi-v7a;arm64-v8a;x86
+ANDROID_ABI_INCLUDE=armeabi-v7a;arm64-v8a;x86;x86_64
```

Comment:
Shouldn't we just drop `x86` then?

Reply:
Hmm, worth exploring, I'll test removing it locally

Reply:
modifying this field had no effect actually, we'll have to update 
https://github.com/status-im/status-mobile/blob/develop/nix/mobile/android/release.nix#L24

Reply:
Probably even higher - Jenkinsfile/Makefile.